polyA/_runners.py

```diff
@@ -198,6 +197,21 @@ def run_full(
             else:
                 stops_matrix.append(tr.stop)
 
+    # Save original alignments for SODA viz
+    subfam_alignments = list(subfamily_sequences_matrix)
+    chrom_alignments = list(chromosome_sequences_matrix)
+    subfam_align_matrix: Dict[str, int] = {}
+    for i in range(1, len(subfamily_matrix)):
+        subfam = subfamily_matrix[i]
+        start = starts_matrix[i]
+        stop = stops_matrix[i]
+        # account for overlaps
```

Comment:
What would overlap? I thought we only had one continuous run per row.

Reply:
This is referring to the fact that we can have an alignment from the same subfamily overlap. So we might have an alignment to some subfamily from pos 3920-4120 and from 3940-4200. However, now that I'm looking at this bit, I realized we don't need it since I took care of mapping a position back to it's original alignment in the collapse_matrices function